keyboards/jopr/jopr.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    MX_A0, MX_B0, MX_C0, MX_D0, MX_E0, MX_F0, MX_G0, MX_H0, MX_I0, MX_J0, MX_A6, MX_B6, MX_C6, MX_D6, MX_E6, MX_F6, MX_G6, MX_H6, \
```

Comment:
Generally the standard for these LAYOUT macros is to use `Kxxx`, where the first digit is the row, and the next two are the columns (in hexadecimal).

Reply:
I tried to be consistent with info.json, which itself is, as suggested, based on the silkscreen text on the PCB. If being consistent with info.json is irrelevant, I'll happily change it.

Reply:
Changed anyway for now.

Reply:
I don't think the label in the info.json has any relation to the layout macro arguments. AFAIK it is not used anywhere and is just for ease of creating/reading the file, for now at least.

I see a lot of info.json files created using KLE and they mostly just use the legends that it assigns to them.